model/ftn/wmesmfmd.ftn

```diff
@@ -2826,6 +3049,44 @@
         enddo
       endif
 !
+! 2.h Set ESMF import grid corner coordinates, only for coupled with mediator
+!
+      if (med_present) then 
```

Comment:
Instead of using med_preset for adding corners we want to check the grid/mesh type then add corners as needed. This can always be on.

Reply:
I think this part of code assumes to work on ESMF grid because all the ESMF calls in this subroutine related with grid and also the name of the subroutine is `CreateImpGrid`. There is also export one `CreateExpGrid`. Do we want to make it on all the time? @JessicaMeixner-NOAA what do you think?